README.md

```diff
@@ -62,3 +62,57 @@ nvm config is saved in `.nvmrc`, set appropriate node version using:
 ```shell
 nvm use
 ```
+
+## Android
```

Comment:
Kunne man her i stedet referere til den readme der er i `lib` så vi ikke må oppdatere to plasser?

Reply:
Hørtes smart ut. Tenker du å lenke til https://static.nrk.no/core-icons/latest/index.html#android da, eller `/lib/readme.md`?

Reply:
Tenker direkte til https://static.nrk.no/core-icons/latest/index.html#android, ja :)

Reply:
Tror ikke det er mulig i core-docs p.t. men kanskje mulig å bygge noe inn der. i utgangspunktet blir ikke `./README.md` med ut til static, så en link dit fra `./lib/readme.md` vil nok feile.